File: server/withssafy/src/main/java/com/ssafy/withssafy/entity/Message.java
package com.ssafy.withssafy.entity;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "tbl_message")
@Getter
@NoArgsConstructor
public class Message {
    @Id @GeneratedValue
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "u_to_id")
    private User u_to;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "u_from_id")
    private User u_from;

    @Column
    private String content;

    @Column
    private String send_dt;

    @Builder
    public Message(Long id, User u_from, User u_to, String content, String send_dt){
        this.id = id;
        this.u_from = u_from;
        this.u_to = u_to;
        this.content = content;
        this.send_dt = send_dt;
    }
}

File: application/WithSSAFY/app/src/main/java/com/ssafy/withssafy/src/dto/Message.kt
package com.ssafy.withssafy.src.dto

data class Message(
    val content: String,
    val id: Int,
    val send_dt: String,
    val u_fromId: Int,
    val u_toId: Int
){
    constructor(content:String,id:Int,u_fromId:Int,u_toId: Int):this(content, id, "", u_fromId, u_toId)
}